packages/core/getBusyTimes.ts

```diff
@@ -140,6 +150,9 @@ export async function getBusyTimes(params: {
         // doing this allows using the map later to remove the ranges from calendar busy times.
         delete bookingSeatCountMap[bookedAt];
       }
+      if (rest.uid === rescheduleUid) {
+        return aggregate;
```

Comment:
return early, as this is the original rescheduled booking and should not be included in busy times